src/Microsoft.Health.Fhir.Core/Features/Search/Registry/ISearchParameterStatusDataStore.cs

```diff
@@ -12,6 +14,8 @@ public interface ISearchParameterStatusDataStore
     {
         Task<IReadOnlyCollection<ResourceSearchParameterStatus>> GetSearchParameterStatuses();
 
+        Task<ResourceSearchParameterStatus> GetSearchParameterStatus(Uri searchParameterUri, CancellationToken cancellationToken);
```

Comment:
Looks like this is not used?

Reply:
I can revert from this PR if it makes sense, I was it to load a single status with the $configure-sort operation

Reply:
Done